drivers/bus/pci/fdo.c

```diff
@@ -111,12 +139,6 @@ FdoEnumerateDevices(
                    PciConfig.VendorID,
                    PciConfig.DeviceID);
 
-            if (PciConfig.VendorID == 0 && PciConfig.DeviceID == 0)
```

Comment:
General question: Is a device with VendorID != 0 but DeviceID == 0 valid?

Reply:
It's valid